extension/src/views/gradleDaemons/models/DaemonStatus.ts

```diff
@@ -0,0 +1,7 @@
+export enum DaemonStatus {
+    IDLE = 0,
```

Comment:
Why not just directly make the value of the enum as the output of `--status` command?

If we change it to string, will we get rid of the invocation like `const status = DaemonStatus[statusString as keyof typeof DaemonStatus];`?